src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageSearchMetadataContextInfo.cs

```diff
@@ -67,7 +67,7 @@ public static PackageSearchMetadataContextInfo Create(IPackageSearchMetadata pac
                 PackageDetailsUrl = packageSearchMetadata.PackageDetailsUrl,
                 PackagePath =
                     (packageSearchMetadata as LocalPackageSearchMetadata)?.PackagePath ??
-                    (packageSearchMetadata as ClonedPackageSearchMetadata)?.PackagePath,
+                    (packageSearchMetadata as PackageSearchMetadataBuilder.ClonedPackageSearchMetadata)?.PackagePath,
```

Comment:
Any idea what made this necessary?

Reply:
ClonedPackageSearchMetadata is a class defined in PackageSearchMetadataBuilder. 
I can do 
```using static NuGet.Protocol.Core.Types.PackageSearchMetadataBuilder```

Reply:
That type existed prior to this PR. Can the whole file just be reverted?